frontend/src/scenes/heatmaps/scenes/heatmap/heatmapLogic.ts

```diff
@@ -199,6 +201,11 @@ export const heatmapLogic = kea<heatmapLogicType>([
             },
         ],
     }),
+    subscriptions(({ actions }) => ({
```

Comment:
value subscriptions are slower than action listeners

could this be a listener instead?